src/test/ui/issues/issue-75907_b.stderr

```diff
@@ -2,8 +2,30 @@ error[E0532]: cannot match against a tuple struct which contains private fields
   --> $DIR/issue-75907_b.rs:9:9
    |
 LL |     let Bar(x, y, z) = make_bar();
-   |         ^^^ constructor is not visible here due to private fields
+   |         ^^^
+   |
+note: constructor is not visible here due to private fields
+  --> $DIR/issue-75907_b.rs:9:16
+   |
+LL |     let Bar(x, y, z) = make_bar();
+   |                ^  ^ private field
+   |                |
+   |                private field
```

Comment:
While looking at this again, I think we could make this slightly different, but it is not introduced by you:

```suggestion
   |         ^^^     -  - private field
   |         |       |
   |         |       private field
   |         constructor not visible here due to private fields
```